### PR TITLE
Improve PHPStan Configuration

### PR DESCRIPTION
**Pull request by simPod**
Moving some parameters inside config file

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .phpcs-cache
-composer.phar
```
Comment (mcg-web) [inline .gitignore]: Why removing this rule?

File: phpstan.neon.dist [hunk 2]
```diff
@@ -1,3 +1,12 @@
+parameters:
+    level: 1
+
+    paths:
+        - %currentWorkingDirectory%/src
+        - %currentWorkingDirectory%/tests
+
+    tmpDir: %currentWorkingDirectory%/var/phpstan
```
Comment (mcg-web) [inline phpstan.neon.dist]: why change phpstan tmp dir? the default sys tmp seems to be good choose, no?

File: .gitignore (hunk 1 above)
Comment (simPod) [reply .gitignore]: It is not part of this project and is rather user specific. 

It should be eventually [ignored globally](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)

File: phpstan.neon.dist (hunk 2 above)
Comment (simPod) [reply phpstan.neon.dist]: I see, I'll revert it

File: .gitignore (hunk 1 above)
Comment (mcg-web) [reply .gitignore]: maybe we should add this to the documentation? cause composer documentation push to use composer.phar direct in the project itself. but I agree this is no a be deal.

**Pull request merged**